run_test.sh

```diff
@@ -46,23 +48,37 @@ find generated -type f -not \( -name "*.expected" -or -name "__init__.py" \) -de
 
     # Create virtualenv
     if [[ -z $SKIP_CLEAN ]] || [[ ! -e $VENV ]]; then
+        rm -rf $VENV
         python3 --version
         python3 -m pip --version
         python3 -m virtualenv $VENV
     fi
     source $VENV/bin/activate
     if [[ -z $SKIP_CLEAN ]] || [[ ! -e $VENV ]]; then
         python3 -m pip install setuptools
+        python3 -m pip install --ignore-requires-python grpc-stubs
         python3 -m pip install git+https://github.com/python/mypy.git@985a20d87eb3a516ff4457041a77026b4c6bd784
     fi
 
+    # preparing isolated folder for testing mypy per python version
+    rm -rf generated_3.5
+    cp -R generated generated_3.5
+
+    rm -rf generated_2.7
+    cp -R generated generated_2.7
+    find generated_2.7 -name '*grpc.pyi' -exec rm {} \;
+    find generated_2.7 -name '*grpc.pyi.expected' -exec rm {} \;
```

Comment:
Don't need this stuff any more I think?
W/ the CI refactor, each run of `run_test.sh` only covers a single $PY_VER_MYPY_TARGET - and therefore should only need one `generated` directory.

Reply:
Ok, I will replace with condition. Not it is possible